ui/app/AppLayouts/Onboarding/views/SeedPhraseInputView.qml

```diff
@@ -21,6 +22,28 @@ OnboardingBasePage {
 
     signal seedValidated()
 
+    Connections {
+        target: onboardingModule
```

Comment:
could you please use the instance from OnboardingStore?

Reply:
@alexandraB99 fixed